cookbook/workflows/sync/loop_steps_workflow.py

```diff
@@ -0,0 +1,87 @@
+from typing import List
+
+from agno.agent import Agent
+from agno.tools.duckduckgo import DuckDuckGoTools
+from agno.tools.hackernews import HackerNewsTools
+from agno.workflow.v2 import Loop, Step, Workflow
+from agno.workflow.v2.types import StepOutput
+
+# Create agents for research
+research_agent = Agent(
+    name="Research Agent",
+    role="Research specialist",
+    tools=[HackerNewsTools(), DuckDuckGoTools()],
+    instructions="You are a research specialist. Research the given topic thoroughly.",
+    markdown=True,
+)
+
+content_agent = Agent(
+    name="Content Agent",
+    role="Content creator",
+    instructions="You are a content creator. Create engaging content based on research.",
+    markdown=True,
+)
+
+# Create research steps
+research_hackernews_step = Step(
+    name="Research HackerNews",
+    agent=research_agent,
+    description="Research trending topics on HackerNews",
+)
+
+research_web_step = Step(
+    name="Research Web",
+    agent=research_agent,
+    description="Research additional information from web sources",
+)
+
+content_step = Step(
+    name="Create Content",
+    agent=content_agent,
+    description="Create content based on research findings",
+)
+
+# End condition function
+
+
+def research_evaluator(outputs: List[StepOutput]) -> bool:
```

Comment:
I don't think it makes sense to get a list here. That implies that the steps run independently in the loop, but they should run sequentially. Like normal workflow execution.